### PR TITLE
Update data.js

### PR DESCRIPTION
**Pull request by tanishqj-19**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -8,4 +8,13 @@ const peopleChoiceData = [
...
         favSeriesID: '62560-mr-robot'
     },
-]
\ No newline at end of file
+    {
+        username: 'tanishqj-19',
+        favAnime: 'Jujutsu Kaisen',
+        favAnimeID: '95479',
+        favMovie: 'Fight Club',
+        favMovieID: '50-fight-club',
```
Comment (aditeyaS) [inline data.js]: incorrect

Comment (aditeyaS) [reply data.js]: It's 550 not 50

**Review by aditeyaS — CHANGES_REQUESTED** [review]
Update id of fight club

File: data.js (hunk 1 above)
Comment (tanishqj-19) [reply data.js]: Sorry for my mistake aditya I have updated it.

**Review by aditeyaS — APPROVED** [review]

**Pull request merged**